modules/gapi/test/common/gapi_core_tests_inl.hpp

```diff
@@ -684,8 +684,8 @@ TEST_P(SumTest, AccuracyTest)
     cv::Size sz_in = std::get<1>(param);
     auto tolerance = std::get<3>(param);
     auto compile_args = std::get<4>(param);
-    //initMatrixRandU(std::get<0>(param), sz_in, std::get<2>(param));
-    initMatsRandN(std::get<0>(param), sz_in, std::get<2>(param)); //TODO: workaround trying to fix SumTest failures
+    initMatrixRandU(std::get<0>(param), sz_in, std::get<2>(param));
+    //initMatsRandN(std::get<0>(param), sz_in, std::get<2>(param)); //TODO: workaround trying to fix SumTest failures
```

Comment:
Does this change actually fix the problem with Sum?

Reply:
No. Removed obsolete comment.